addons/im_livechat/tests/test_user_livechat_username.py

```diff
@@ -81,3 +81,34 @@ def test_user_livechat_username_reactions(self):
                 "write_date": fields.Datetime.to_string(john.partner_id.write_date),
             },
         )
+
+    def test_user_livechat_username_in_store(self):
+        dog_expert = self.env["im_livechat.expertise"].create({"name": "dog"})
+        john = self._create_operator("fr_FR", expertises=dog_expert)
+        operator = self._create_operator("en_US")
+
+        john.livechat_username = "ELOPERADOR"
+        operator.group_ids = [
+            fields.Command.link(self.env.ref("im_livechat.im_livechat_group_user").id),
+            fields.Command.link(self.env.ref("base.group_user").id),
```

Comment:
Didn't we still fix `im_livechat_group_user` in this version so that it properly implies `group_user`?